docs/source/use_cases/perform_quality_control_checks/verify_bam_id.rst

```diff
@@ -0,0 +1,95 @@
+Verify Bam Id
+===========
+
+.. toctree::
+   :maxdepth: 2
+
+.. contents::
+
+This pipeline tests a set of reads for contamination.  It takes a set of specified ReadGroupSets
+of reads to test and statistics on reference allele frequencies for SNPs with a single alternative
+from a specified set of VariantSets and combines these to produce an estimate of the amount of
+contamination.
```

Comment:
This sentence really runs. Would be easier to read as:

===
This pipeline tests a set of reads for contamination.  It takes as input:

* a set of ReadGroupSets to test
* statistics on reference allele frequencies for SNPs with a single alternative from a set of VariantSets

and combines these to produce an estimate of the amount of contamination.
===